.gitignore

```diff
@@ -1 +1,9 @@
 .vscode/*
+
+# NPM things
+package.json
+package-lock.json
+node_modules/
+
+# Yarn things
+yarn.lock
```

Comment:
lmaooo ok ok this is the closest you've ever come to submitting something resulting in me _immediately_ thinking ABSOLUTELY NOT 

buuuut since the end user experience doesn't require any evil NPM summoning and it's all happening on github's infrastructure and it stays away from _my_ computer i GUESS i'll let it slide this time ;)

Reply:
Yeah, I was conflicted about this addition too, I only justified it to myself that if someone really needed to lint things and had, I don't know, only a notepad as an editor (not recommended), they would have a way to lint things.

As it is some developers only, and we actively discourage any npms to crop up inside the actual application via the gitignore I decided to include in the pull request for you to give it a look.